### DISCUSSION TITLE
Getting appcheck working on expo sdk 45

### DISCUSSION THREAD

**Discussion by skjorrface** [invertase/react-native-firebase#6311] [Q&A]
As title says, I'm struggling to get appcheck working on expo development client, android, sdk 45. I followed every step in the official documentation, created a debug token, inserted it into eas.json, development profile, 
<img width="638" alt="Schermata 2022-06-14 alle 23 39 02" src="https://user-images.githubusercontent.com/9011806/173693840-95587f13-8643-4467-a0bc-06632e5f08a3.png">

FIREBASE_APP_CHECK_DEBUG_TOKEN, but as I enable appcheck enforcement mode from firebase console, requests simply stop working. What am I doing wrong? P.S. I already tried removing android folder, resetting metro and expo cache and rebuilding again but it didn't help and of course I'm using a real device to do the tests.

**Comment by atanaskanchev** [comment]
Same here, I have been unable to get App Check working in a development build.

**Comment by mikehardy** [comment]
As a non-expo user I'm not sure how to support it at all. App Check config is non-trivial as is integrating firebase in general.
Perhaps a first step would be to list out all steps taken starting with `expo init` or whatever the "hey let's create a fresh/empty project" command is for expo? Then as a non-expo user I may be able to poke through it still and see what's going wrong.

**Reply by skjorrface** [reply]
I just made a release build and uploaded it to Apple testflight for inner tests... It flawlessly worked. I will do the same for android. So the problem seems to be related to the development environment only.
